### PR TITLE
docs: gpstop new option --host

### PR DESCRIPTION
**Pull request by mkiyama**
See related PR https://github.com/greenplum-db/gpdb/pull/3910

### CODE REVIEW

File: gpdb-doc/dita/utility_guide/admin_utilities/gpstop.xml [hunk 1]
```diff
@@ -57,6 +60,23 @@
                     <pd>Optional. The master host data directory. If not specified, the value set
                         for <codeph>$MASTER_DATA_DIRECTORY</codeph> will be used.</pd>
                 </plentry>
+                <plentry>
+                    <pt>--host <varname>host_name</varname></pt>
+                    <pd>If segment mirroring is enabled, the utility shuts down the Greenplum
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpstop.xml]: Confused about this - from the dev story I thought this stopped segments regardless of whether mirroring was enabled (which would necessitate some different docs/warnings on usage here).  I also thought the code we didn't consider cases where primary and mirror were on the same host.

File: gpdb-doc/dita/utility_guide/admin_utilities/gpstop.xml [hunk 2]
```diff
@@ -57,6 +60,23 @@
...
+                        Database segment instances on the specified host. The primary segment
+                        instances on the host are shut down and the associated mirror segment
+                        instances are promoted to primary segments. Mirror segment instances on the
+                        host are shut down.</pd>
+                    <pd>The segment instances are not shut down and utility returns an error in
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpstop.xml]: utility -> the utility

File: gpdb-doc/dita/utility_guide/admin_utilities/gpstop.xml [hunk 3]
```diff
@@ -57,6 +60,23 @@
...
+                        these cases:<ul id="ul_at2_kk3_ccb">
+                            <li>Segment mirroring is not enabled for the system.</li>
+                            <li>The master or standby master is on the host.</li>
+                            <li>Both a primary segment instance and its mirror are on the host.</li>
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpstop.xml]: This seems to contradict the earlier description, which describes the utility shutting down primary, allowing mirror to be promoted, and then shutting down the mirror.

**Comment by mkiyama** [comment]
updated --host description based on review comments.

@dyozie - The description is based on the text in the gpstop_help file in the dev PR https://github.com/greenplum-db/gpdb/pull/3910

File: gpdb-doc/dita/utility_guide/admin_utilities/gpstop.xml (hunk 1 above)
Comment (cjcjameson) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpstop.xml]: Ah, we had further discussions and development with @skahler-pivotal . This feature:
 - Fails if segment mirroring is not enabled
 - Fails if primary and corresponding mirror are on the same host.

Maybe it's clearer if we remove "If segment mirroring is enabled," and just let the bullet point below express this point. The flag-feature isn't about mirroring, per se, it's more about hardware being broken/slow, and wanting to take it out of service while keeping the cluster running (which requires mirrors, but it's a eventual necessity, not a primary concern).

**Comment by mkiyama** [comment]
@skahler-pivotal 
Could you check the note I added in the last commit.
I has information about using gpseginstall to restore the segment instances.

**Comment by skahler-pivotal** [comment]
@mkiyama that should be gprecoverseg instead of gpseginstall

**Comment by mkiyama** [comment]
thanks @skahler-pivotal 
corrected name to gprecoverseg.

**Pull request merged**